test/TokenTimelock.js

```diff
@@ -29,26 +28,26 @@ contract('TokenTimelock', function ([_, owner, beneficiary]) {
   })
 
   it('cannot be released just before time limit', async function () {
-    await increaseTime(moment.duration(0.99, 'year').asSeconds())
+    await increaseTimeTo(this.releaseTime - duration.seconds(3))
```

Comment:
As `testrpc` is run on localhost the lag is usually up to 1s. Due to time rounding,  subtracting a single second is not enough to have a robust behaviour but 3s seems to be safe enough. 

Obviously, the situation may change if someone decides to run tests across `testrpc` instance connected. But, to be honest, I don't expect it to be a common practice.